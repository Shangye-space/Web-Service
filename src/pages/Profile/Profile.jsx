import React from 'react';
import styles from './Profile.module.scss';

const Account = () => {
    return (
        <div className="account-style">
            <div className="account-main-style">
                <br></br>
            </div>

            <div className="account-main-style">
                {' '}
                {/* FIRST Block*/}
                <div className="container account-main-style">
                    <div className="row py-1 ">
                        {/* Column 0 -blank*/}
                        <div className="col-md-2 col-sm-6"></div>

                        {/* Column 1 - Личные Данные */}
                        <div className="col-md-3 col-sm-6">
                            <ul className="list-unstyled text-center">
                                <div>
                                    <a href="#!" className="account-white">
                                        ЛИЧНЫЕ ДАННЫЕ
                                    </a>
                                </div>
                            </ul>
                        </div>

                        {/* Column 2 - Заказы*/}
                        <div className="col-md-2 col-sm-6">
                            <ul className="list-unstyled text-center">
                                <div>
                                    <a href="#!" className="account-white">
                                        ЗАКАЗЫ
                                    </a>
                                </div>
                            </ul>
                        </div>

                        {/* Column 3 - Список Желани*/}
                        <div className="col-md-3 col-sm-6">
                            <ul className="list-unstyled text-center">
                                <div>
                                    <a href="#!" className="account-white">
                                        СПИСОК ЖЕЛАНИЙ
                                    </a>
                                </div>
                            </ul>
                        </div>

                        {/* Column 0 -blank */}
                        <div className="col-md-2 col-sm-6"></div>

                        <div className="py-3"></div>
                    </div>
                </div>
            </div>

            <div className="container account-style2">
                {' '}
                {/* SECOND Block*/}
                <div className="acoount-style2 col-md-2 col-sm-6">
                    <br></br>
                </div>
                <div className="row">
                    {/* Column 0 Photo*/}
                    <div className="col-md-2 col-sm-6">
                        <h4 className=" main-color">
                            <img
                                className="img"
                                alt=""
                                src="https://i.ibb.co/NKjNb0b/Photo1.jpg"
                            ></img>
                        </h4>
                    </div>

                    {/* Column 1 Name*/}
                    <div className="col-md-4 col-sm-6">
                        <h4 className="black ">JEGOR LAPTEV</h4>
                        <ul className="list-unstyled">
                            <li className="list-unstyled">
                                <a href="#!" className="acoount-gray-thin">
                                    Edit Photo...
                                </a>
                            </li>
                        </ul>
                    </div>

                    {/* Column 2 Contacts*/}
                    <div className="col-md-5 col-sm-6">
                        <h4 className="main-color account">Contact with us</h4>
                        <ul className="list-unstyled">
                            <br></br>
                            <li className="list-unstyled">
                                <a href="#!" className="acoount-gray-thin">
                                    help@shanye.space
                                </a>
                            </li>
                            <li className="list-unstyled">
                                <a href="#!" className="acoount-gray-thin">
                                    (+372) 53078808
                                </a>
                            </li>
                            <li className="list-unstyled">
                                <a href="#!" className="acoount-gray-thin">
                                    skype: shangye
                                </a>
                            </li>
                        </ul>
                    </div>
                </div>
                <hr align="center" width="800" size="2" color="#E49881" />{' '}
                {/* Line */}
                <div className="acoount-bottom">
                    <br></br>
                </div>
            </div>

            <div className="container account-style2">
                {' '}
                {/* THIRD Block*/}
                <div className="row">
                    {/* Name */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Full Name</h6>
                        <input
                            type="text"
                            name="fullnameMain"
                            value="Jegor Laptev"
                            className="black-s"
                        ></input>
                    </div>

                    {/* Email */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Email Adress</h6>
                        <input
                            type="text"
                            name="emailMain"
                            value="j.laptev@gmail.com"
                            className="black-s"
                        ></input>
                    </div>

                    {/* Phone */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Phone Number</h6>
                        <input
                            type="text"
                            name="phoneMain"
                            value="+372 53078809"
                            className="black-s"
                        ></input>
                    </div>

                    {/* Country */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Country</h6>
                        <input
                            type="text"
                            name="countryMain"
                            value="Estonia"
                            className="black-s"
                        ></input>
                    </div>

                    {/* Language */}
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">Language</h6>
                        <select>
                            <option value="volvo">English</option>
                            <option value="saab">Русский</option>
                            <option value="opel">Eesti</option>
                        </select>
                    </div>
                </div>
                {/* Line */}
                <div className="acoount-style2">
                    <br></br>
                    <hr align="center" width="800" size="2" color="#E49881" />
                    <br></br>
                </div>
                {/* FOURTH Block */}
                <div className="acoount-style2">
                    <h4 className="black">Change Password</h4>
                    <br></br>
                </div>
                <div className="row">
                    {/* Column 1 */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Old Password</h6>
                        <input
                            type="text"
                            name="oldPassword"
                            value=" **********"
                            className="black-s"
                        ></input>
                    </div>

                    {/* Column 2 */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">New Password</h6>
                        <input
                            type="text"
                            name="newPassword"
                            value=""
                            className="black-s"
                        ></input>
                    </div>

                    {/* Column 3 */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Confirm Password</h6>
                        <input
                            type="text"
                            name="confirmPassword"
                            value=""
                            className="black-s"
                        ></input>
                    </div>

                    {/* Column 4 */}
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">
                            {' '}
                            <br></br>
                        </h6>
                        <input type="submit" value="Confirm password"></input>
                    </div>
                </div>
                {/* Line */}
                <div className="acoount-style2">
                    <br></br>
                    <hr align="center" width="800" size="2" color="#E49881" />
                    <br></br>
                </div>
                {/* FIVETH Block */}
                <div className="acoount-style2">
                    <h4 className="black">Shipping Information</h4>
                    <br></br>
                </div>
                <div className="row">
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">First Name</h6>
                        <input
                            type="text"
                            name="fname"
                            value="Jegor Laptev"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6">
                        <h6 className="black">Last Name</h6>
                        <input
                            type="text"
                            name="fname"
                            value="j.laptev@gmail.com"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-6 col-sm-6"> {/* Blank */}</div>
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">Street</h6>
                        <input
                            type="text"
                            name="street"
                            value="Moisa"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">House, Flat</h6>
                        <input
                            type="text"
                            name="house"
                            value="46 - 343"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">Zip</h6>
                        <input
                            type="text"
                            name="zip"
                            value="12616"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6"></div> {/* Blank */}
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">Country</h6>
                        <input
                            type="text"
                            name="country"
                            value="Eesti"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">State</h6>
                        <input
                            type="text"
                            name="state"
                            value="Harjumaa"
                            className="black-s"
                        ></input>
                    </div>
                    <div className="col-md-3 col-sm-6">
                        <br></br>
                        <h6 className="black">Town</h6>
                        <input
                            type="text"
                            name="town"
                            value="Tallinn"
                            className="black-s"
                        ></input>
                    </div>
                </div>
                {/* Line */}
                <div className="acoount-style2">
                    <br></br>
                    <hr align="center" width="800" size="2" color="#E49881" />
                    <br></br>
                </div>
                {/* SIXTH Block */}
                <div className="acoount-style2">
                    <h4 className="black">Diactivate Account</h4>
                    <ul className="list-unstyled">
                        <li className="list-unstyled">
                            <p className="acoount-gray-thin">
                                The monkey-rope is found in all whalers; but it
                                was only in the Pequod that the monkey and his
                                holder were ever tied together. This improvement
                                upon the original usage was introduced by no.
                            </p>
                            <a href="#!" className="acoount-gray-thin2">
                                Diactivete Your Account
                            </a>
                        </li>
                    </ul>
                    <br></br>
                </div>
            </div>
        </div>
    );
};

export default Account;
