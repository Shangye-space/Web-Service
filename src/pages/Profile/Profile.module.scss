.account-main-style {
    background-color: #282828;
    color: white;
}
.main-color {
    color: #e49881;
    font-family: sans-serif;
}

.black {
    color: black;
    font-family: sans-serif;
    font-weight: bold;
}

.black-s {
    color: rgb(95, 95, 95);
    font-family: sans-serif;
}

.account {
    color: #e49881;
    font-family: sans-serif;
    font-weight: bold;
}

.account-white {
    color: white;
    font-family: sans-serif;
    font-weight: bold;
}

.accoun-white-thin {
    color: white;
    font-family: sans-serif;
}

.acoount-gray-thin {
    color: rgb(63, 63, 63);
    font-family: sans-serif;
}

.acoount-gray-thin2 {
    color: #e49881;
    font-family: sans-serif;
}

.img {
    max-width: 100%;
    height: auto;
    width: 100%;
}

.account-style {
    background-color: white;
    color: black;
}

.account-style2 {
    background-color: white;
    color: black;
}
