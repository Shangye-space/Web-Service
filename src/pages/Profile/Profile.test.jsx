import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Profile from './Profile.jsx';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Profile />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<Profile />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
