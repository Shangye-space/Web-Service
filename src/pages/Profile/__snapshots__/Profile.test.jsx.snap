// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="account-style"
>
  <div
    className="account-main-style"
  >
    <br />
  </div>
  <div
    className="account-main-style"
  >
     
    <div
      className="container account-main-style"
    >
      <div
        className="row py-1 "
      >
        <div
          className="col-md-2 col-sm-6"
        />
        <div
          className="col-md-3 col-sm-6"
        >
          <ul
            className="list-unstyled text-center"
          >
            <div>
              <a
                className="account-white"
                href="#!"
              >
                ЛИЧНЫЕ ДАННЫЕ
              </a>
            </div>
          </ul>
        </div>
        <div
          className="col-md-2 col-sm-6"
        >
          <ul
            className="list-unstyled text-center"
          >
            <div>
              <a
                className="account-white"
                href="#!"
              >
                ЗАКАЗЫ
              </a>
            </div>
          </ul>
        </div>
        <div
          className="col-md-3 col-sm-6"
        >
          <ul
            className="list-unstyled text-center"
          >
            <div>
              <a
                className="account-white"
                href="#!"
              >
                СПИСОК ЖЕЛАНИЙ
              </a>
            </div>
          </ul>
        </div>
        <div
          className="col-md-2 col-sm-6"
        />
        <div
          className="py-3"
        />
      </div>
    </div>
  </div>
  <div
    className="container account-style2"
  >
     
    <div
      className="acoount-style2 col-md-2 col-sm-6"
    >
      <br />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-2 col-sm-6"
      >
        <h4
          className=" main-color"
        >
          <img
            alt=""
            className="img"
            src="https://i.ibb.co/NKjNb0b/Photo1.jpg"
          />
        </h4>
      </div>
      <div
        className="col-md-4 col-sm-6"
      >
        <h4
          className="black "
        >
          JEGOR LAPTEV
        </h4>
        <ul
          className="list-unstyled"
        >
          <li
            className="list-unstyled"
          >
            <a
              className="acoount-gray-thin"
              href="#!"
            >
              Edit Photo...
            </a>
          </li>
        </ul>
      </div>
      <div
        className="col-md-5 col-sm-6"
      >
        <h4
          className="main-color account"
        >
          Contact with us
        </h4>
        <ul
          className="list-unstyled"
        >
          <br />
          <li
            className="list-unstyled"
          >
            <a
              className="acoount-gray-thin"
              href="#!"
            >
              help@shanye.space
            </a>
          </li>
          <li
            className="list-unstyled"
          >
            <a
              className="acoount-gray-thin"
              href="#!"
            >
              (+372) 53078808
            </a>
          </li>
          <li
            className="list-unstyled"
          >
            <a
              className="acoount-gray-thin"
              href="#!"
            >
              skype: shangye
            </a>
          </li>
        </ul>
      </div>
    </div>
    <hr
      align="center"
      color="#E49881"
      size="2"
      width="800"
    />
     
    <div
      className="acoount-bottom"
    >
      <br />
    </div>
  </div>
  <div
    className="container account-style2"
  >
     
    <div
      className="row"
    >
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Full Name
        </h6>
        <input
          className="black-s"
          name="fullnameMain"
          type="text"
          value="Jegor Laptev"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Email Adress
        </h6>
        <input
          className="black-s"
          name="emailMain"
          type="text"
          value="j.laptev@gmail.com"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Phone Number
        </h6>
        <input
          className="black-s"
          name="phoneMain"
          type="text"
          value="+372 53078809"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Country
        </h6>
        <input
          className="black-s"
          name="countryMain"
          type="text"
          value="Estonia"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          Language
        </h6>
        <select>
          <option
            value="volvo"
          >
            English
          </option>
          <option
            value="saab"
          >
            Русский
          </option>
          <option
            value="opel"
          >
            Eesti
          </option>
        </select>
      </div>
    </div>
    <div
      className="acoount-style2"
    >
      <br />
      <hr
        align="center"
        color="#E49881"
        size="2"
        width="800"
      />
      <br />
    </div>
    <div
      className="acoount-style2"
    >
      <h4
        className="black"
      >
        Change Password
      </h4>
      <br />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Old Password
        </h6>
        <input
          className="black-s"
          name="oldPassword"
          type="text"
          value=" **********"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          New Password
        </h6>
        <input
          className="black-s"
          name="newPassword"
          type="text"
          value=""
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Confirm Password
        </h6>
        <input
          className="black-s"
          name="confirmPassword"
          type="text"
          value=""
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
           
          <br />
        </h6>
        <input
          type="submit"
          value="Confirm password"
        />
      </div>
    </div>
    <div
      className="acoount-style2"
    >
      <br />
      <hr
        align="center"
        color="#E49881"
        size="2"
        width="800"
      />
      <br />
    </div>
    <div
      className="acoount-style2"
    >
      <h4
        className="black"
      >
        Shipping Information
      </h4>
      <br />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          First Name
        </h6>
        <input
          className="black-s"
          name="fname"
          type="text"
          value="Jegor Laptev"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <h6
          className="black"
        >
          Last Name
        </h6>
        <input
          className="black-s"
          name="fname"
          type="text"
          value="j.laptev@gmail.com"
        />
      </div>
      <div
        className="col-md-6 col-sm-6"
      >
         
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          Street
        </h6>
        <input
          className="black-s"
          name="street"
          type="text"
          value="Moisa"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          House, Flat
        </h6>
        <input
          className="black-s"
          name="house"
          type="text"
          value="46 - 343"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          Zip
        </h6>
        <input
          className="black-s"
          name="zip"
          type="text"
          value="12616"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      />
       
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          Country
        </h6>
        <input
          className="black-s"
          name="country"
          type="text"
          value="Eesti"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          State
        </h6>
        <input
          className="black-s"
          name="state"
          type="text"
          value="Harjumaa"
        />
      </div>
      <div
        className="col-md-3 col-sm-6"
      >
        <br />
        <h6
          className="black"
        >
          Town
        </h6>
        <input
          className="black-s"
          name="town"
          type="text"
          value="Tallinn"
        />
      </div>
    </div>
    <div
      className="acoount-style2"
    >
      <br />
      <hr
        align="center"
        color="#E49881"
        size="2"
        width="800"
      />
      <br />
    </div>
    <div
      className="acoount-style2"
    >
      <h4
        className="black"
      >
        Diactivate Account
      </h4>
      <ul
        className="list-unstyled"
      >
        <li
          className="list-unstyled"
        >
          <p
            className="acoount-gray-thin"
          >
            The monkey-rope is found in all whalers; but it was only in the Pequod that the monkey and his holder were ever tied together. This improvement upon the original usage was introduced by no.
          </p>
          <a
            className="acoount-gray-thin2"
            href="#!"
          >
            Diactivete Your Account
          </a>
        </li>
      </ul>
      <br />
    </div>
  </div>
</div>
`;
